aten/src/ATen/native/mps/MPSGraphVenturaOps.h

```diff
@@ -16,6 +48,26 @@ typedef NS_ENUM(NSUInteger, MPSGraphResizeNearestRoundingMode)
     MPSGraphResizeNearestRoundingModeRoundToEven       =  4L,
     MPSGraphResizeNearestRoundingModeRoundToOdd        =  5L,
 };
+
+- (MPSGraphTensor * _Nonnull) convolution3DWithSourceTensor:(MPSGraphTensor * _Nonnull) source
+                                    weightsTensor:(MPSGraphTensor * _Nonnull) weights
+                                       descriptor:(MPSGraphConvolution3DOpDescriptor * _Nonnull) descriptor
+                                                      name:(NSString * _Nullable) name;
+
+- (MPSGraphTensor * _Nonnull) convolution3DDataGradientWithIncomingGradientTensor:(MPSGraphTensor * _Nonnull) incomingGradient
+                                                          weightsTensor:(MPSGraphTensor * _Nonnull) weights
+                                                            outputShape:(MPSShape * _Nonnull) outputShape
+                                           forwardConvolutionDescriptor:(MPSGraphConvolution3DOpDescriptor * _Nonnull) forwardConvolutionDescriptor
+                                                                   name:(NSString * _Nullable) name;
+
+- (MPSGraphTensor * _Nonnull) convolution3DWeightsGradientWithIncomingGradientTensor:(MPSGraphTensor * _Nonnull) incomingGradient
```

Comment:
These need to be defined outside the macro.

Reply:
Thanks a lot @kulinseth this was exactly the problem that caused the build failure. It is now fixed and passes the basic tests.